### PR TITLE
Development namespace access control

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Adds access control for the development namespace

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

- Jira cards
    - [OPS-6](https://airqoteam.atlassian.net/browse/OPS-6)

**_HOW DO I TEST OUT THIS PR?_**
- Get the dev-user access token
- Login as the dev-user
- Try to access resources of any other namespace

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 
- n/a

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**
- n/a

**_ARE THERE ANY RELATED PRs?_**
- n/a

### CODE REVIEW

**Review by NoahNsimbe — APPROVED** [review]
LGTM

File: k8s/clusters/staging/access-control/dev-user.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  name: dev-user
+  namespace: development
+
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: dev-admin-role
+  namespace: development
+rules:
```
Comment (123MwanjeMike) [inline k8s/clusters/staging/access-control/dev-user.yaml:13]: Giving admin all the permissions in the development namespace

File: k8s/clusters/staging/access-control/dev-user.yaml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  kind: ClusterRole
+  name: view
+
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: dev-dashboard-view-clusterrole
+rules:
+- apiGroups: [""]
```
Comment (123MwanjeMike) [inline k8s/clusters/staging/access-control/dev-user.yaml:53]: Giving access to only the default api group

File: k8s/clusters/staging/access-control/dev-user.yaml [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+  resources: ["pods/attach", "pods/exec", "pods/portforward", "pods/proxy", "services/proxy"]
+  verbs: ["get", "list", "watch", "create"]
```
Comment (123MwanjeMike) [inline k8s/clusters/staging/access-control/dev-user.yaml:55]: Permissions on the resources needed to access the k8s dashboard

**Review by Baalmart — COMMENTED** [review]
All looks good to me, thanks @123MwanjeMike . Just one small inquiry....during the rename/creation of the respective cluster directories......wasn't the one for **production** created?

Just want to make sure I fully appreciate ongoings here.

Thanks!

**Comment by 123MwanjeMike** [comment]
> All looks good to me, thanks @123MwanjeMike . Just one small inquiry....during the rename/creation of the respective cluster directories......wasn't the one for **production** created?
> 
> Just want to make sure I fully appreciate ongoings here.
> 
> Thanks!

It was create as seen [here](https://github.com/airqo-platform/AirQo-api/tree/ft-dev-accessControl/k8s/clusters/production) with it's contents @Baalmart

**Review by Baalmart — APPROVED** [review]
LGTM, thanks @123MwanjeMike

**Pull request merged**